pkg/asset/internal/templates.go

```diff
@@ -156,6 +156,36 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 `)
 
+	KubeletApproverTemplate = []byte(`
+# WARNING: This deployment should not be used in production clusters as it will
+# automatically approve every kubelet CSRs without making any kind of validation
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kubelet-approver
+  namespace: kube-system
+  labels:
+    app: k8s-app
```

Comment:
This still needs to be fixed. See other examples in this file: https://github.com/kubernetes-incubator/bootkube/blob/master/pkg/asset/internal/templates.go#L580